Lesson7/ArithmeticAverageTest.java

```diff
@@ -0,0 +1,61 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ArithmeticAverageTest {
+
+    ArithmeticAverage aa = new ArithmeticAverage();
+    static int a;
+    static int b;
+
+    @BeforeAll
+    public static void init() {
+
+    }
+
+    @BeforeEach
+    public static void initEach() {
+        a = -5;
+        b = 12;
+    }
+
+    @Test
+    public void isArithmeticAver()
+    {
+        this.initEach();
+        double expected = 15;
+        double actual = aa.theArithmeticAverageOfAllNumbers(a, b);
+        Assertions.assertEquals(expected, actual, "Expected is Fail");
+    }
+
+    @BeforeEach
```

Comment:
why you need BeforeEach twice, pls use one beforeEach for common context (precondition which used in all tests)